model_connection.go

```diff
@@ -60,11 +60,11 @@ func (c *ConnectionParameters) UnmarshalYAML(unmarshaller func(interface{}) erro
 	if err := unmarshaller(&temp); err != nil {
 		return fmt.Errorf("failed to JSON unmarshal data (%w)", err)
 	}
-	unserializedData, err := connectionParametersSchema.UnserializeType(temp)
+	unserializedData, err := connectionParametersSchema.UnserializeType(temp) //nolint:all
 	if err != nil {
 		return fmt.Errorf("failed to unserialize data (%w)", err)
 	}
-	c = &unserializedData
+	c = &unserializedData //nolint:all
```

Comment:
Disabling the linter completely for those two lines is a bit heavy handed -- not that I object -- but, it would be good if there were a comment explaining why this is currently necessary/appropriate (and maybe it should cite issue #18?).

Reply:
I also had that thought. Do I just add the url address to that issue at those lines in the code?

Reply:
Sure; although, a small amount of "local" explanation would be good, too, so that the reader doesn't _have_ to copy the URL to a browser and fetch the page, etc.

I would use something like this for line 67:
```
// TODO:  This assignment has no effect!  We disable the linter's objection to it pending investigation -- see https://github.com/arcalot/arcaflow-lib-kubernetes-go/issues/18
```

Reply:
haha the linter doesn't like `TODO`s in the source code

Reply:
Yeah, that's probably a Good Thing™.  There should be a way to mark things in the code which should _not_ be merged into the upstream (I usually use `FIXME` for that), but we also need a (hopefully rarely used...) way to mark things in the code which need to be addressed later and, therefore, necessarily need to be merged (which is what I would use `TODO` for).

The IDE and/or linter should be configured to highlight these as concerns, but it shouldn't cause the build to fail.  C'est la vie.  Omitting the token is good enough.